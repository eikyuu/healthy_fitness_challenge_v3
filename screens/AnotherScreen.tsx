import { useNavigation } from "@react-navigation/native";
import { Button, StatusBar, Text, View } from "react-native";
import AsyncStorage from '@react-native-async-storage/async-storage';
import { useEffect } from 'react';

export default function AnotherScreen() {

  return (
    <View>
      <Text>Open up App.tsx to start working on your app!</Text>
    </View>
  );
}