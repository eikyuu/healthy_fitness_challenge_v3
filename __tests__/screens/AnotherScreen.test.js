import React from 'react';
import renderer from 'react-test-renderer';
import AnotherScreen from '../../screens/AnotherScreen';

describe('<AnotherScreen />', () => {
  it('has 1 child', () => {
    const tree = renderer.create(<AnotherScreen/>).toJSON();
    expect(tree.children.length).toBe(1);
  });


  it('renders correctly', () => {
    const tree = renderer.create(<AnotherScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });

});