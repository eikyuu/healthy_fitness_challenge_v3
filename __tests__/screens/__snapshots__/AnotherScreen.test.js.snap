// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AnotherScreen /> renders correctly 1`] = `
<View>
  <Text>
    Open up App.tsx to start working on your app!
  </Text>
</View>
`;
